### ISSUE TITLE
question about finetune FVD

### ISSUE THREAD

**Issue by 523997931** [alibaba/animate-anything#86]
thanks for your work, I use the scripts to finetune SVD, I don't konw if it can directly use scheduler.add_noise to create noise latent, than use unet to denoise, and I use one video to overfit test the scripts, I find all is welll, except the color, the overfit model generate video color is deeper than the input model, I don't know why

**Comment by zhenghao977** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by 523997931** [comment]
> thanks for your work, I use the scripts to finetune SVD, I don't konw if it can directly use scheduler.add_noise to create noise latent, than use unet to denoise, and I use one video to overfit test the scripts, I find all is welll, except the color, the overfit model generate video color is deeper than the input model, I don't know why

and a silly question.... I saw the source code of stable video diffusion in diffusers, and I don't find it multiply the vae scale factor(0.18125) when it encode the image to latent, but I saw in training code, it added, I don't know why.
